specs/ceph_installer_wrapper.adoc

```diff
@@ -54,8 +54,45 @@ Tendrl API provides an interface to create a Ceph cluster.
 Sample JSON
 
 ----------
-POST '/api/v1/CreateCluster'
-
+POST '/api/1.0/CreateCluster'
+
+{
+  "sds_type": "ceph",
+  "sds_version": "10.2.5",
+  "name": "MyCluster",
+  "node_configuration": {
+    "3a95fd96-876d-439a-a64d-70332c069aaa": {
+      "role": "ceph/mon"
+    },
+    "3943fab1-9ed2-4eb6-8121-5a69499c4568": {
```

Comment:
osd node details should contain `public_network` and `cluster_network` values as well. These would be provided while create-cluster flow from UI and should be passed down here through api layer. Same applies for mon node details as well.